Modules/faulthandler.c

```diff
@@ -253,6 +332,32 @@ faulthandler_dump_traceback_py(PyObject *self,
     Py_RETURN_NONE;
 }
 
+static PyObject *
+faulthandler_dump_c_stack_py(PyObject *self,
+                               PyObject *args, PyObject *kwargs)
+{
+    static char *kwlist[] = {"file", NULL};
+    PyObject *file = NULL;
+
+    if (!PyArg_ParseTupleAndKeywords(args, kwargs,
+        "|p:dump_c_stack", kwlist,
+        &file))
+        return NULL;
```

Comment:
```suggestion
    if (!PyArg_ParseTupleAndKeywords(args, kwargs, 
                                     "|p:dump_c_stack",
                                     kwlist, &file))
 	{
    	return NULL;
    }
```

For this one I never know whether PEP-7 tells me to put `{` on the same line as `)` or not but the example seems to do so.

Reply:
I would rather be consistent with the rest of the file. Usage of `PyArg_ParseTupleAndKeywords` omits braces, so let's keep it that way (maybe worth fixing in another PR though!)

Reply:
I'm ok with the consistency. Just that it's hard to spot this when reviewing parts of big files. I'll trust you for where to put PEP-7 things and where not to! Although, I would stick to a single style for new functions.

Reply:
Really, the best style for new functions would be to just use the clinic. (Speaking of which, is `faulthandler` thread safe?)

Reply:
I don't know about the thread-safety of faulthandler but I wouldn't really count on it considering it's really really low-level. But maybe it is and the magic works! (I've really rarely used `faulthandler`).